### PR TITLE
librtasevent: include endian.h for beXXtoh macros

### PR DESCRIPTION
**Pull request by grobian**
In particular using musl libc, without this include, the calls to
be16toh/be32toh result in a compiler warning about an undefined symbol.
Since in musl these are actually implemented by macros, the symbols
remain undefined in the final shared library, making the library
unusable.

Issue: https://github.com/ibm-power-utilities/librtas/issues/10
Signed-off-by: Fabian Groffen <grobian@gentoo.org>

### CODE REVIEW

**Review by hegdevasant — APPROVED** [review]

**Comment by grobian** [comment]
Is there anything you expect me to do?  I cannot merge the code after your approval...

**Comment by hegdevasant** [comment]
> Is there anything you expect me to do? I cannot merge the code after your approval...

@tyreld is the maintainer. He has to review/merge the code.  For now, no action required from your side.

-Vasant

**Comment by nathanlynch** [comment]
lgtm. @tyreld ?

**Review by nathanlynch — APPROVED** [review]

**Comment by tyreld** [comment]
Fixed by commit 6447b24d9f8d892e750b9517f7c136855967ea61

**Pull request closed without merging**
